openrouteservice/src/main/java/heigit/ors/routing/graphhopper/extensions/flagencoders/ORSAbstractFlagEncoder.java

```diff
@@ -22,6 +22,7 @@
 
 public abstract class ORSAbstractFlagEncoder extends AbstractFlagEncoder {
 	private final static Logger logger = LoggerFactory.getLogger(ORSAbstractFlagEncoder.class);
+	protected static double MEAN_SPEED;
```

Comment:
If you declare the getMeanSpeed method as abstract, it doesn't have a body and so this variable isn't needed